providers/fab/src/airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -234,6 +243,7 @@ class FabAirflowSecurityManagerOverride(AirflowSecurityManagerV2):
         (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG_RUN),
         (permissions.ACTION_CAN_READ, RESOURCE_ASSET),
         (permissions.ACTION_CAN_READ, RESOURCE_ASSET_ALIAS),
+        (permissions.ACTION_CAN_READ, RESOURCE_BACKFILL),
```

Comment:
Other permissions than READ need to be added to the OP role. This is something that has been forgotten for AssetAlias, I'll comment on the PR

Reply:
added. thanks!